build.go

```diff
@@ -38,18 +39,30 @@ func Build(entryResolver EntryResolver, dependencyService DependencyService, pro
 		logger.Title(context.BuildpackInfo)
 
 		logger.Process("Resolving Nginx Server version")
-		logger.Candidates(context.Plan.Entries)
 
-		entry := entryResolver.Resolve(context.Plan.Entries)
+		priorities := []interface{}{
+			"BP_NGINX_VERSION",
+			"buildpack.yml",
```

Comment:
Can you add `"buildpack.toml"` that's used as a version source in detect here as well for completeness? Other than that LGTM